Source/WebKit/UIProcess/Cocoa/VideoFullscreenManagerProxy.mm

```diff
@@ -598,6 +598,17 @@ - (BOOL)prefersStatusBarHidden
     }
     [view layer].frame = CGRectMake(0, 0, initialSize.width(), initialSize.height());
     [view setContextID:videoLayerID];
+
+    [interface->captionsLayer() removeFromSuperlayer];
+    [[view layer] addSublayer:interface->captionsLayer()];
+
+    [CATransaction begin];
+    [CATransaction setDisableActions:YES];
+    interface->captionsLayer().zPosition = FLT_MAX;
+    [interface->captionsLayer() setAnchorPoint:CGPointZero];
+    [interface->captionsLayer() setBounds:CGRectMake(0, 0, initialSize.width(), initialSize.height())];
+    [CATransaction commit];
```

Comment:
This is fine for now, but I this should all move into VideoFullscreenInterfaceAVKit eventually.